eclair-core/src/test/scala/fr/acinq/eclair/router/RouteCalculationSpec.scala

```diff
@@ -1662,6 +1662,14 @@ class RouteCalculationSpec extends AnyFunSuite with ParallelTestExecution {
     val fees = routes.map(_.fee)
     assert(fees.forall(_ == fees.head))
   }
+
+  test("can't relay if fee is not sufficient") {
+    val g = DirectedGraph(List(
+      makeEdge(1L, a, b, 1000 msat, 7000),
+    ))
+
+    val Failure(_) = findRoute(g, a, b, 10000000 msat, 10000 msat, numRoutes = 3, routeParams = DEFAULT_ROUTE_PARAMS.copy(isRelay = true), currentBlockHeight = 400000)
```

Comment:
nit: we can be a little more precise:
```
assert(findRoute(g, a, b, 10000000 msat, 10000 msat, numRoutes = 3, routeParams = DEFAULT_ROUTE_PARAMS.copy(isRelay = true), currentBlockHeight = 400000) === Failure(RouteNotFound)
```

And I would also add the positive case for completeness:

```
assert(findRoute(g, a, b, 10000000 msat, 100000 msat, numRoutes = 3, routeParams = DEFAULT_ROUTE_PARAMS.copy(isRelay = true), currentBlockHeight = 400000).isSuccess)
```